app/components/Documentation/EditDocumentation.tsx

```diff
@@ -137,6 +160,47 @@ const EditDocumentation: React.FC<EditDocumentationProps> = ({
     });
   };
 
+  const onClickArticle = (id: string): void => {
+    if (
+      typeof setViewArticleDispatch === 'function' &&
+      typeof setEditArticleDispatch === 'function'
+    ) {
+      setViewArticleDispatch(id);
+      setEditArticleDispatch({
+        id: id,
+        parentId: '1',
+        isEditArticle: false,
+      });
+    }
+  };
+  const onClickPrevArticle = (): void => {
+    const prevArrayId = treeArray.filter(
+      (item) => item?.treeIndex == currentArrayIndex - 1,
+    )[0]?.node.id;
+    onClickArticle(prevArrayId);
+  };
+
+  const onClickNextArticle = (): void => {
+    const nextArrayId = treeArray.filter(
```

Comment:
use method find

combine functions onClickNextArticle  and onClickPrevArticle  into one, use a closure

```
const onClickArticle = (typeButton) => () => {
  ...
}
```